library/src/test/java/bio/terra/pfb/utils/PfbJsonComparator.java

```diff
@@ -0,0 +1,23 @@
+package bio.terra.pfb.utils;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.TextNode;
+import java.util.Comparator;
+
+class PfbJsonComparator implements Comparator<JsonNode> {
+  @Override
+  public int compare(JsonNode o1, JsonNode o2) {
+    if (o1.equals(o2)) {
+      return 0;
+    }
+    if ((o1 instanceof TextNode) && (o2 instanceof TextNode)) {
+      String s1 = ((TextNode) o1).asText();
+      String s2 = ((TextNode) o2).asText();
+
+      if (s1.equalsIgnoreCase(s2)) {
+        return 0;
+      }
+    }
+    return 1;
```

Comment:
Ok, I've actually removed this -- I was using it for debugging to try to pinpoint where pyPFB output differs from the java-pfb output.

Reply:
Good idea!  I think renaming it to something that doesn't imply conformity to `Comparator` interface makes perfect sense, eg: `compareIgnoringOrdering`; this will distinguish it from the more rigid requirements of `Comparator`.

Reply:
Sounds good!